pages/index.html

```diff
@@ -2,11 +2,64 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../css/indexStyle.css"/>
+  <link rel="icon"  href="../assets/yanzi5.ico">
+  <title>Yanxian Zhang</title>
 </head>
 <body>
-  
+  <div class="headMessage">
+    <nav class="menu">
+      <ul>
+        <a href="index.html" target="_top"> HOME </a>
+        <a href="about.html" target="_top"> About </a>
+        <a href="portfolio.html" target="_top"> Protfolio </a>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="homepage">
+    <div class="selfie">
+      <img src="../assets/self.png" alt="miranda-selfie">
+    </div>
+    <div class="contact">
+      <h1> Yanxian Zhang</h1>
+      <h2> Miranda</h2>
+      <h4>Software Development Student at</h4>
+      <h4> <a class="ada_link" href="https://adadevelopersacademy.org/" target="_blank">Ada Developers Academy</a></h4>
+      <div class="contact_pics">
+        <a href="https://www.linkedin.com/in/yanxian-zhang-4b1a92239/" target="_blank">
+          <img src="../assets/link.png" alt="linkedin"> 
+        </a>
+        <a href="mailto:yanxianzhang99@gmail.com">
+          <img src="../assets/email1.png" alt="email">
+        </a>
+        <a href="https://github.com/yxzhang88" target="_blank"> 
+          <img src="../assets/github1.png" alt="github" > 
+        </a>
+      </div>
+    </div>
+
+    <div class="selfIntro"> 
+      <div class="introText"> 
+        <p> 
+          Hello, I'm Miranda. Currently I am a full-time student study at Ada Developers Academy which is a non-profit, tuition-free coding school for women and gender expansive adults. I learned include Python, SQL, Flask, HTML/CSS, JavaScript, and Computer Science fundamentals. </p>
+        <p>
+          As a new immigrants, in these 3 years, 'language' learning is priority in my life constantly. Not only study the English to communicate with people, also study the programming language and try to communicate with computers.  
+        </p>
+        <p>
+          I'm a person who likes to continuously trying new things. As Jack in the Titanic said, <em><u>"I figure life's a gift and I don't intend on wasting it. You don't know what hand you're gonna get dealt next. You learn to take life as it comes at you. To make each day count".</em></u>
```

Comment:
Generally, tags should open and close in order if they are right next to each other. Since `em` was opened first, it should close first. The ordering is causing errors.